packages/applications-service-api/src/utils/sort-applications.merge.js

```diff
@@ -24,8 +24,14 @@ const sortApplications = (applications, sort = '+ProjectName') => {
 			return applications.sort(sortByNumberKeyInObject(key, direction));
 	}
 };
+
+const notNullOrUndefined = (value) => !value && value !== 0;
```

Comment:
I had to think about this.  Is this
 (value !== null) || (value === undefined)
or is it
!(value === null || value === undefined) 

I get these results with the current code:
notNullOrUndefined() -> true
notNullOrUndefined(undefined) -> true
notNullOrUndefined(null) -> true
notNullOrUndefined(0) -> false
notNullOrUndefined(1) -> false

Reply:
I've changed the name to be more useful `isNullOrUndefined`. 

It's more so like `value === null || value === undefined || value === false`
The results you got seem to be as expected, essentially it should return true for anything falsy (null, undefined, false), except 0. This is just because a valid stage is 0 (draft stage).